Scripts/phylotol.py

```diff
@@ -3,20 +3,8 @@
 import os, sys
 import Utilities
 
-def get_parameters():
-	PathtoFiles = ''					##Path to folders ncbiFiles, BlastFiles, allOGFiles and the OGs of interest file
-	testPipelineList = ''			##Name of the file with the OGs of interest -  this file needs to be in the same location as your datafile folders
-	listTaxaInterest = ''
-	blastCutOff = 1e-15				##Blast e-value cutoff (also depends on how blasts were run)
-	seqLenCompCutOff = 20			##We remove very short sequences during the Taxon step - this is the amino acid length cutoff
-	tooSimCutOff = 0.98				##Sequences are removed if they are more similar than this cutoff (default = 0.98)
-	guidanceIter = 'n'				##number of Guidance iterations for sequences removal. the default is running until no bad scores remain (default = n)
-	seqcutoff = 0.4 				##During guidance, taxa are removed if their score is below this cutoff (default as of 11/2014 = .35)
-	colcutoff = 0.4					##During guidance, taxa are removed if their score is below this cutoff (default as of 11/2014 = .40)
-	rescutoff = 0.4					##During guidance, residues are removed if their score is below this cutoff (default as of 03/2017 = .40)  # MAC---I added this v3
-	concatAlignment = ''					##Do you want to remove paralogs and generate an alignment for concatenation? (y/n)  # MAC---I added this v3
```

Comment:
These were like "the default" values for phylotol. Not really necessary. The user always has to provide a parameters file